CONTRIBUTING.md

```diff
@@ -18,10 +18,11 @@ We encourage all forms of contributions to SMARTS, not limited to:
 
 ## Setting up your dev environment
 
-In addition to following the setup steps in the README, you'll need to install the [dev] dependencies.
+In addition to following the setup steps in the README, you'll need to install the [dev] dependencies and install the pre-commit hooks.
 
 ```bash
 pip install -e .[dev]
+pre-commit install
```

Comment:
This should not be needed because it will be installed with `[dev]`.

Reply:
ah with that i meant the `pre-commit install` command

Reply:
it is, this is to setup our pre-commit hooks for use not the actual install

Reply:
Oh, sorry, yes!